src/state/learnocaml_data.mli

```diff
@@ -178,6 +178,12 @@ module Exercise: sig
     val by_status:
       Token.Set.t -> assignments -> (status * Token.Set.t) list
 
+    (* Merges all changes from [theirs] and [ours], based on [ancestor]. [ours]
```

Comment:
By privileging "ours", you choose a last-arrived-get-the-prize strategy right?

Reply:
yes, as mentionned in the commit notes. This happens for conflicting changes on same exercise + same student.
If we choose the alternative (refuse conflicting changes), we need to add a way to report the partial error to the client.

Reply:
That's the good design choice I think. I just wanted to check I correctly understood.